builtin/docker/plugin.pb.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Not sure why these are in this PR, but it's ok!

Reply:
originally i had changes to the ecr protobuf but ended up not needing that. this is just a remnant from that!